_search-plugins/hybrid-search.md

```diff
@@ -569,4 +569,454 @@ The response contains the matching documents and the aggregation results:
+
+Sort by `doc_price` and `search_after` condition applied on it
+
+```json
+GET /my-nlp-index/_search?search_pipeline=nlp-search-pipeline
+{
+  "query": {
+    "hybrid": {
+      "queries": [
+        {
+          "term": {
+            "category": "permission"
+          }
+        },
+        {
+          "bool": {
+            "should": [
+              {
+                "term": {
+                  "category": "editor"
+                }
+              },
+              {
+                "term": {
+                  "category": "statement"
+                }
+              }
+            ]
+          }
+        }
+      ]
+    }
+  },
+  "sort":[
+     {
+        "_id": {
+          "order": "desc"   
+        }
+     } 
+  ],
+  "search_after":[200]
+}
+```
+{% include copy-curl.html %}
+
+The response contains the matching documents that are after `200` and sorted by `doc_price` in decreasing order:
+
+```json
+{
+    "took": 8,
+    "timed_out": false,
+    "_shards": {
+        "total": 3,
+        "successful": 3,
+        "skipped": 0,
+        "failed": 0
+    },
+    "hits": {
+        "total": {
+            "value": 4,
+            "relation": "eq"
+        },
+        "max_score": 0.5,
+        "hits": [
+            {
+                "_index": "my-nlp-index",
+                "_id": "6yaM4JABZkI1FQv8AwoM",
+                "_score": null,
+                "_source": {
+                    "category": "permission",
+                    "doc_keyword": "workable",
+                    "doc_index": 4976,
+                    "doc_price": 100
+                },
+                "sort": [
+                    100
+                ]
+            },
+            {
+                "_index": "my-nlp-index",
+                "_id": "7iaM4JABZkI1FQv8AwoN",
+                "_score": null,
+                "_source": {
+                    "category": "editor",
+                    "doc_index": 9871,
+                    "doc_price": 30
+                },
+                "sort": [
+                    30
+                ]
+            }
+        ]
+    }
+}
+```
+
+Sort by `id` and `search_after` condition applied on it 
+
+```json
+GET /my-nlp-index/_search?search_pipeline=nlp-search-pipeline
+{
+  "query": {
+    "hybrid": {
+      "queries": [
+        {
+          "term": {
+            "category": "permission"
+          }
+        },
+        {
+          "bool": {
+            "should": [
+              {
+                "term": {
+                  "category": "editor"
+                }
+              },
+              {
+                "term": {
+                  "category": "statement"
+                }
+              }
+            ]
+          }
+        }
+      ]
+    }
+  },
+  "sort":[
+     {
+        "_id": {
+          "order": "desc"   
+        }
+     } 
+  ],
+  "search_after":["7yaM4JABZkI1FQv8AwoN"]
```

Comment:
if `7yaM4JABZkI1FQv8AwoN` is a user provided id can we use something simpler, where order is obvious, like tiny, small, medium, large, and you pick everything after medium up to large

Reply:
_id is generated by system so OpenSearch can apply sort on it. tiny, small, medium, large are text and OpenSearch does not support sorting on text fields.